### PR TITLE
Use twilio-ruby next-gen library

### PR DESCRIPTION
**Pull request by joliveros**

### CODE REVIEW

**Comment by joliveros** [comment]
Hi @acamino,
Would you be so kind to review this PR?

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,5 @@
 /log/*
 !/log/.keep
 /tmp
+
+.env
```
Comment (acamino) [inline .gitignore]: 👍

File: app/controllers/twilio_controller.rb [hunk 2]
```diff
@@ -2,12 +2,14 @@
 require 'sanitize'
 require 'csv'
 
+TWILIO_API_HOST = 'api.twilio.com'
```
Comment (acamino) [inline app/controllers/twilio_controller.rb]: Seems like `TWILIO_API_HOST` is not used. Could you please double check, if this is the case, please remove it. We don't want to leave dead code. 😉

File: app/controllers/twilio_controller.rb [hunk 3]
```diff
@@ -89,19 +91,27 @@ def broadcast
   def start_call_record
     phone_number = params[:phone_number]
 
-    @client.account.calls.create(
-      from: '+15005550006',
+    @client.calls.create(
+      from: @twilio_number,
       to: phone_number,
       url: request.base_url + '/broadcast/record'
```
Comment (acamino) [inline app/controllers/twilio_controller.rb]: I know you didn't introduce this change. Did you consider using string interpolation here?

File: app/controllers/twilio_controller.rb [hunk 4]
```diff
@@ -2,12 +2,14 @@
 require 'sanitize'
```
Comment (acamino) [inline app/controllers/twilio_controller.rb]: Rails autoload takes care of loading those dependencies (_requires_) automatically. Could you remove them please? -- I know you didn't put them here, we're just cleaning up this controller.

File: app/controllers/twilio_controller.rb [hunk 5]
```diff
@@ -2,12 +2,14 @@
...
 require 'csv'
 
+TWILIO_API_HOST = 'api.twilio.com'
+
 class TwilioController < ApplicationController
-  before_action :set_client_and_number, only: [:start_call_record, :broadcast_send, :fetch_recordings]
+  before_action :set_client_and_number, only: [:start_call_record, :broadcast_send, :fetch_recordings, :conference]
```
Comment (acamino) [inline app/controllers/twilio_controller.rb]: 💯

File: README.md [hunk 6]
```diff
@@ -1,56 +1,78 @@
+<a href="https://www.twilio.com">
```
Comment (acamino) [inline README.md]: `README` has evolved over time. Would you mind using this one https://github.com/TwilioDevEd/eta-notifications-rails/blob/master/README.md as reference for updating this one. Any suggestions are welcome.

**Comment by acamino** [comment]
Good job @joliveros!  Would you mind addressing the comments I left?

File: app/controllers/twilio_controller.rb (hunk 2 above)
Comment (acamino) [inline app/controllers/twilio_controller.rb]: Thanks!

**Comment by acamino** [comment]
Thanks @joliveros & @mcelicalderon!

**Pull request merged**